### DISCUSSION TITLE
AMD显卡的玄学

### DISCUSSION THREAD

**Discussion by hooke007** [hooke007/mpv_PlayKit#38] [mpv 通用]
因为主力机是iu+n卡，也没注意amd的特殊情况（直到我拿mbp的5600m试了试...），虽然记得似乎原帖确实有amd用户反馈过一些奇奇怪怪的问题。这里做个记录

**Comment by hooke007** [comment]
mac下 `vo=libmpv` 的性能表现简直一坨shi，但是又没得选，装win10快乐使用 `vo=gpu` ——


```
gpu-context=winvk
hwdec=d3d11va-copy
```

**Comment by natural-harmonia-gropius** [comment]
显卡驱动太烂了，去amd社区经常性抱怨两句说不定就排上日程了
